charts/internal/shoot-system-components/charts/aws-custom-route-controller/templates/role.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: extensions.gardener.cloud:provider-aws:aws-custom-route-controller
+  namespace: kube-system
+rules:
+  - apiGroups:
```

```suggestion
- apiGroups:
```
Nit: Use same indentation as in other files